odh-dashboard/base/params.env

```
@@ -0,0 +1 @@
+odh-dashboard-image=quay.io/modh/odh-dashboard-container@sha256:67fab181cacb1c41d342897f535c0880802b8ee9b6596fc3689847339bafb86e
```

`67fab181cacb1c41d342897f535c0880802b8ee9b6596fc3689847339bafb86e` digest corresponds to `v1.17.0-9`, however this information will be in the deployer. For dynamic images, the default value should be a generic tag:

```suggestion
odh-dashboard-image=quay.io/modh/odh-dashboard-container:latest
```